src/common/config/metadata/metadatalist.go

```diff
@@ -121,6 +121,7 @@ var (
 
 		{Name: common.ProjectCreationRestriction, Scope: UserScope, Group: BasicGroup, EnvKey: "PROJECT_CREATION_RESTRICTION", DefaultValue: common.ProCrtRestrEveryone, ItemType: &ProjectCreationRestrictionType{}, Editable: false},
 		{Name: common.ReadOnly, Scope: UserScope, Group: BasicGroup, EnvKey: "READ_ONLY", DefaultValue: "false", ItemType: &BoolType{}, Editable: false},
+		{Name: common.DisableAnonymous, Scope: UserScope, Group: BasicGroup, EnvKey: "DISABLE_ANONYMOUS", DefaultValue: "false", ItemType: &BoolType{}, Editable: false},
```

Comment:
To make it straightforward,  can we use AllowAnonymousUser and default value is true?

Reply:
Do you mean allow sounds more positive than disable? ;-) I am ok with the change; and for now, the default value of DisableAnonymous is false.

Reply:
I think we should keep consistent with the previous experience, allow anonymous. With this feature, the user can disable it via configuration.